bitnami/rabbitmq/README.md

```diff
@@ -263,41 +263,29 @@ extraConfiguration: |
   log.console.formatter = json
 ```
 
-### Recover the cluster from complete shutdown
+### How to Avoid Deadlocked Deployments After a Cluster-Wide Restart
 
-> IMPORTANT: Some of these procedures can lead to data loss. Always make a backup beforehand.
+RabbitMQ nodes assume their peers come back online within five minutes (by default). When the `OrderedReady` pod management policy is used with a readiness probe that implicitly requires a fully booted node, the deployment can deadlock:
 
-The RabbitMQ cluster is able to support multiple node failures but, in a situation in which all the nodes are brought down at the same time, the cluster might not be able to self-recover.
+- Kubernetes will expect the first node to pass a readiness probe
+- The readiness probe may require a fully booted node
+- The node will fully boot after it detects that its peers have come online
+- Kubernetes will not start any more pods until the first one boots
 
-This happens if the pod management policy of the statefulset is not `Parallel` and the last pod to be running wasn't the first pod of the statefulset. If that happens, update the pod management policy to recover a healthy state:
+The following combination of deployment settings avoids the problem:
```

Comment:
Hi,
  Is there any kind of process that user could perform to once he is in this situation ?.
   I mean, I am not sure if the user has already deployed without the parameters you indicate here he could avoid the issue by just upgrading the deployment with the parameter or if he may need to execute other steps.

Reply:
@michaelklishin one of our clusters got into this state, but when I tried to change the podManagementStrategy to parallel, in my testing k8s does not allow you to allow you to change that field on an existing statefulset.   So to change this strategy we would have to delete our statefulset and recreate